### PR TITLE
Feature/get in touch

### PR DESCRIPTION
**Pull request by raeioul**

### CODE REVIEW

File: src/pages/containers/Footer.js [hunk 1]
```diff
@@ -58,9 +59,11 @@ class Footer extends Component {
         <div className="social-icons-container">
           <img src={logo} alt="logo" />
           <div className="social-icons-item">{this.mapIcons()}</div>
-          <button onClick={this.toggleHidden} className="get-in-touch-button">
-            GET IN TOUCH !
-          </button>
+
+          <Link to="/get-in-touch" className="get-in-touch-button">
+            {' '}
```
Comment (diegolanda) [inline src/pages/containers/Footer.js]: needs reformat

File: src/pages/containers/Footer.js [hunk 2]
```diff
@@ -58,9 +59,11 @@ class Footer extends Component {
...
+            GET IN TOUCH !{' '}
```
Comment (diegolanda) [inline src/pages/containers/Footer.js]: same

File: src/pages/containers/GetInTouch.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import TopNavBar from './TopNavBar';
+import Footer from './Footer';
+import ContactUs from './ContactUs';
+import webdesign from '../assets/images/book-design.png';
```
Comment (diegolanda) [inline src/pages/containers/GetInTouch.js]: should be camel cased

File: src/pages/containers/GetInTouch.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+          get_in_touch,
+        })
+      );
+  }
+
+  getStyles = () => {
+    const { get_in_touch } = this.state;
+    const { backgroundRepeat, backgroundImage, backgroundSize } = get_in_touch;
+    const styles = {
+      backgroundRepeat: backgroundRepeat,
```
Comment (diegolanda) [inline src/pages/containers/GetInTouch.js]: Not entirely sure I want to store these values on a json file, but looks good for now

File: src/pages/containers/GetInTouch.js (hunk 3 above)
Comment (raeioul) [reply src/pages/containers/GetInTouch.js]: Fixed.

**Pull request merged**